### PR TITLE
docs: how to walkthrough

### PR DESCRIPTION
**Pull request by 0xGorilla**

### CODE REVIEW

File: docs/src/content/how-to/automation_vault.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Create your first Automation Vault
+
+An Automation Vault serves as the home for your on-chain automations. 
+
+Within your vault, you can activate various automation networks (for example, Gelato) and select specific tasks they are authorized to perform.
```
Comment (ashitakah) [inline docs/src/content/how-to/automation_vault.md:5]: Should we add keep3r as it is our sponsor?

File: docs/src/content/how-to/automation_vault.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+In order to pay for your onchain automation, you can deposit ETH or any ERC20 into your vault.
```
Comment (ashitakah) [inline docs/src/content/how-to/automation_vault.md:7]: now, open relay only work with ETH and gelato is ETH payment, its true that we accept ERC20 but we can work with that.

File: docs/src/content/how-to/automation_vault.md (hunk 1 above)
Comment (0xGorilla) [reply docs/src/content/how-to/automation_vault.md:5]: No need

**Review by ashitakah — DISMISSED** [review]

**Review by ashitakah — APPROVED** [review]
:LFG

**Pull request merged**